src/node.js

```diff
@@ -335,23 +335,27 @@
         scheduleMicrotasks();
     }
 
+    function _tryCallback(callback) {
+      var threw = true;
+      try {
+        callback();
+        threw = false;
+      } finally {
+        if (threw)
+          tickDone();
```

Comment:
Assuming there isn't some weird negative performance impact, wouldn't it be better to use `try {} catch {}` instead? This would avoid the need for `threw`.